content/updates/2021-02/3-Quality-score-in-the-product-export-builder.md

```diff
@@ -0,0 +1,26 @@
+---
+pim_announcement_img: ./img/product-export-filters-on-quality-score.png
+pim_announcement_alt_img: "Product export filters on quality score"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Quality score in the Product Export Builder
```

Comment:
It's the name we use in the documentation. But I agree it can be confusing for new users.
However, we can't just say "export" cause you can both export via API and the export builder.